crates/graphql/src/schema/context.rs

```diff
@@ -37,7 +37,7 @@ use scalars::{
     PublicKey,
 };
 
-use super::prelude::*;
+use super::{dataloaders::nft::MoonrankRank, prelude::*};
```

Comment:
There is already an import block for the dataloaders can you use that?

https://github.com/holaplex/indexer/pull/855/files#diff-b63748f494193303e21412e5a7260eaca25ed37dbcc2b1c914d4d23d602214a0R1